src/sentry/api/bases/organization_events.py

```diff
@@ -427,7 +427,7 @@ def get_event_stats_data(
                 columns = request.GET.getlist("yAxis", _columns)
 
                 if query is None:
-                    query = request.GET.get("query")
+                    query = request.GET.get("query", "")
```

Comment:
Instead of giving this a default value of `""` I think the `SearchResolver` needs to handle `None` for the query string.

Reply:
Hmm we can have it handle that, but typing already expects it to be an empty string, and I also noticed later we do `split_query = f"({query}) AND !event.type:transaction"` 🙃

Reply:
In that case, can we support `None` in the `SearchResolver` as well?

Reply:
Sure thing

Reply:
https://github.com/getsentry/sentry/pull/80990/commits/ca769189b33bfe26a320992dbfcbe8b0202ae2f2